cromwell-backend/src/main/scala/cromwell/backend/model/TaskDescriptor.scala

```diff
@@ -0,0 +1,24 @@
+package cromwell.backend.model
+
+import wdl4s.TaskOutput
+import wdl4s.command.CommandPart
+import wdl4s.values.WdlValue
+
+/**
+  * Represents basic data needed to execute a task in a defined backend.
+ *
+  * @param name Task name.
+  * @param user User who want to execute the task.
+  * @param commandTemplate Arguments for the execution.
+  * @param workingDir Task working directory.
+  * @param outputs Outputs generated by the execution.
+  * @param runtimeAttributes Configuration needed at runtime in order to execute a task.
+  */
+case class TaskDescriptor(name: String,
+                          user: String,
+                          commandTemplate: Seq[CommandPart], //TODO: Should be Wdl Agnostic
```

Comment:
@francares We're actually looking at switching to doing things fully within github & using related ecosystem (e.g. waffle.io for kanban board). It's just one of those things that I never get around to investigating properly and has languished on my todo list for too long.